ddns/cache.py

```diff
@@ -41,18 +41,21 @@ def load(self, file=None):
             file = self.__filename
 
         self.__logger.debug("load cache data from %s", file)
-        if file and path.isfile(file):
-            with open(file, "rb") as data:
-                try:
-                    loaded_data = load(data)
-                    self.clear()
-                    self.update(loaded_data)
-                    self.__time = stat(file).st_mtime
-                    return self
-                except ValueError:
-                    pass
-                except Exception as e:
-                    self.__logger.warning(e)
+        if file:
+            try:
+                with open(file, "r") as data:
+                    try:
```

Comment:
this sencod level `try` is not indeed. just catch all exceptions in the top level